vivisect/remote/server.py

```diff
@@ -55,7 +55,7 @@ def vprint(self, msg):
         return self.server.vprint(msg)
 
     def _fireEvent(self, event, einfo, local=False, skip=None):
-        return self.server._fireEvent(self.wsname, event, einfo, local=local, skip=skip)
+        return self.server._fireEvent(self.chan, event, einfo, local=local, skip=skip)
```

Comment:
fuck.  this breaks all server compatibility, creating a line in the sand.  either all users would have to be before (client and server) or after this PR.
  
i am ok with breaking follow-the-leader because it's always been loosely broken... but this may actually cause people pain. fuck fuck fuck.  we want to transition to using chan instead of "wsname", but we'll have to put a compatibility shim in for a year.